# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/intel-media-sdk/hotfix-cstdint.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

diff -ruN MediaSDK-intel-mediasdk-23.2.2.orig/api/mfx_dispatch/linux/mfxparser.cpp MediaSDK-intel-mediasdk-23.2.2/api/mfx_dispatch/linux/mfxparser.cpp
--- MediaSDK-intel-mediasdk-23.2.2.orig/api/mfx_dispatch/linux/mfxparser.cpp	2023-05-06 03:51:49.000000000 +0200
+++ MediaSDK-intel-mediasdk-23.2.2/api/mfx_dispatch/linux/mfxparser.cpp	2026-10-09 15:47:47.114076369 +0200
@@ -24,6 +24,7 @@
 #include <string.h>
 
 #include <list>
+#include <cstdint>
 
 #include "mfxloader.h"
 
diff -ruN MediaSDK-intel-mediasdk-23.2.2.orig/samples/sample_vpp/src/sample_vpp_frc_adv.cpp MediaSDK-intel-mediasdk-23.2.2/samples/sample_vpp/src/sample_vpp_frc_adv.cpp
--- MediaSDK-intel-mediasdk-23.2.2.orig/samples/sample_vpp/src/sample_vpp_frc_adv.cpp	2023-05-06 03:51:49.000000000 +0200
+++ MediaSDK-intel-mediasdk-23.2.2/samples/sample_vpp/src/sample_vpp_frc_adv.cpp	2026-10-09 16:24:47.441130114 +0200
@@ -21,6 +21,7 @@
 #include "vm/strings_defs.h"
 #include "sample_vpp_frc_adv.h"
 #include <algorithm>
+#include <cstdint>
 
 #ifndef MFX_VERSION
 #error MFX_VERSION not defined
